fujiji-server/migration-scripts/populate.sql

```diff
@@ -0,0 +1,30 @@
+INSERT INTO fujiji_listing
+  (user_id, title, condition, category, city, province_code, image_url, price, description, creation_date)
+VALUES
+  (1839, 'Bed', 'used', 'Bed', 'Winnipeg', 'MB', 'https://source.unsplash.com/gySMaocSdqs/', 80.90, 'Some description here', GETUTCDATE());
+
+INSERT INTO fujiji_listing
+  (user_id, title, condition, category, city, province_code, image_url, price, description, creation_date)
+VALUES
+  (1839, 'Cabinet', 'new', 'Cabinet', 'Vancouver', 'BC', 'https://source.unsplash.com/gySMaocSdqs/', 123, 'Some description here', GETUTCDATE());
+
+INSERT INTO fujiji_listing
+  (user_id, title, condition, category, city, province_code, image_url, price, description, creation_date)
+VALUES
+  (1839, 'Chair', 'refurbished', 'Chair', 'Montreal', 'QC', 'https://source.unsplash.com/gySMaocSdqs/', 10000, 'Some description here', GETUTCDATE());
+
+INSERT INTO fujiji_listing
+  (user_id, title, condition, category, city, province_code, image_url, price, description, creation_date)
+VALUES
+  (1840, 'Sofa', 'used', 'Sofa', 'Winnipeg', 'MB', 'https://source.unsplash.com/gySMaocSdqs/', 99.99, 'Some description here', GETUTCDATE());
```

Comment:
Would it be okay to run this script without having such `user_id`s in the `fujiji_user` table?

Reply:
Nevermind, this was an accidental push I guess